libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/ManeuverProcessor.kt

```diff
@@ -295,6 +298,21 @@ internal object ManeuverProcessor {
         return list
     }
 
+    private fun Maneuver.mapTo(distanceRemaining: Double): Maneuver {
+        return Maneuver(
```

Comment:
nit, we could consider an `internal` copy function

Reply:
Done @LukasPaczos